tests/test_trainer.py

```diff
@@ -151,6 +151,8 @@ def get_regression_trainer(a=0, b=0, double_output=False, train_len=64, eval_len
 
 
 @require_torch
+@require_sentencepiece
```

Comment:
Super nit but can we put those on the actual tests that require one or the other?

Reply:
I use this to skip some test but I agree this is not clean, I'll clean this up in the next PR updating the CI test suite.